web-bundle/src/main/java/com/graphhopper/resources/RouteResource.java

```diff
@@ -75,7 +73,7 @@ public Response doGet(
             @Context HttpServletRequest httpReq,
             @Context UriInfo uriInfo,
             @QueryParam(WAY_POINT_MAX_DISTANCE) @DefaultValue("1") double minPathPrecision,
-            @QueryParam("point") List<GHPointParam> pointParams,
+            @QueryParam("point") @NotNull List<GHPointParam> pointParams,
```

Comment:
@michaz for a moment I was tempted to set this to `@NotEmpty` with a custom error message, but then I thought its more consistent to do this later so its the same for GET&POST. But `@NotNull` cannot hurt, because by default I like to assume that nothing is `null` (for example when I later use the `GHRequest`).